src/main/io/osd.c

```diff
@@ -596,6 +596,13 @@ static void osdDrawSingleElement(uint8_t item)
             break;
         }
 
+        /* Show warning if battery is not fresh */
+        if (!ARMING_FLAG(WAS_EVER_ARMED) && (getBatteryState() == BATTERY_OK)
+              && getBatteryVoltage() < (batteryConfig()->vbatfullcellvoltage * getBatteryCellCount())) {
```

Comment:
should we have `getBatteryAverageCellVoltage()` and use that instead of repeating the logic for `someCellVoltage * getBatteryCellCount()`